Sources/Epoxy/Core/ButtonRow.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  ButtonRow.swift
+//  RakuyoKit
+//
+//  Created by Rakuyo on 2024/5/17.
+//  Copyright © 2024 RakuyoKit. All rights reserved.
+//
+
+import UIKit
+
+import EpoxyCore
+import RAKCore
+
+// MARK: - ButtonRow
+
+/// Replace `UIImageView` in Epoxy component
+///
+/// If you want to extend, consider building your own view with
+/// the help of `ButtonRow.Style`, `ButtonRow.Content` and `ButtonRow.Behaviors`.
+public final class ButtonRow: UIButton {
```

Comment:
I think we can add a `SimplifiedButtonRow` in the future, delete some uncommon attributes, and only provide encapsulation of common methods (for example, only retain the `normal` state)

Or upgrade this class to `AdvancedButtonRow` and leave `ButtonRow` to the simplified version.